i18n/zh-CN/docusaurus-plugin-content-docs/current/deployment/deploy-quick.md

```diff
@@ -68,16 +68,32 @@ drwxrwxr-x 7 hadoop hadoop      4096 Feb 21 10:13 linkis-package // 实际的软
 
 ### 2.2 配置数据库信息
 
+`vim deploy-config/linkis-env.sh`
+
+```shell script
+# 选择linkis业务数据库类型，默认mysql
```

Comment:
batter to  add support version info such as  linkis>=1.4.0

Reply:
Thank you for your feedback. I have made the changes as recommended.